src/async-wrap.cc

```diff
@@ -207,25 +209,23 @@ Local<Value> AsyncWrap::MakeCallback(const Local<Function> cb,
   }
 
   if (ran_init_callback() && !pre_fn.IsEmpty()) {
-    try_catch.SetVerbose(false);
```

Comment:
In the init callback you have to put `this._asyncQueue = {}`. Since it doesn't tap into the `AsyncWrap` class used that long ago to trigger the remaining callbacks. Not a good solution, but haven't had a chance to rethink it.

As for the `setImmediate()`. timers don't currently work ATM. Will require hacking the JS and have been putting that off hoping I'd find a better solution.